File: Roaming_Reciever.pl
#Roaming_Reciver

use 5.022;
use warnings;

my $id = <>;
my $balance = <>;
chomp $balance;

$balance =~ s/\.$//;

say $balance;

1;

# by Tovarisch Trunaev, 2026, Russia, Uray

File: Roaming_Reciever.pas
Program Roaming_Reciever;
Uses
	SysUtils;
Var 
	id, balance : string;
	i : integer;
Begin;
    ReadLn (id);
    ReadLn (balance);
    
    i := Length (balance);
    If (balance[i] = '.') Then
		balance := Copy(balance, 1, i - 1);
    
   	WriteLn (StrToFloat(balance));
end.

{ by Tovarisch Trunaev, 2026, Russia, Uray }
